results/2020/profile.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Profile
+position: 2
+---
+
+# 2.Profile
+
+### What is your gender ?
+
+<Chart id="profile-q-0" pie title={false} />
+
+### What is your age ?
+
+The majority of the respondents have between 20 and 24 years old which represents either junior developers or university graduates. The other category of respondents were people between 25 and 29 years old.
+
+<Chart id="profile-q-1" title={false} />
+
+### Where are you located ?
+
+The majority of the answers were from people located in Morocco with 87.9% of the answers. The rest of the answers were from people located in Europe or other countries
+
+<Chart id="profile-q-2" title={false} />
+
+### What is your highest diploma ?
+
+More than half of the developers that answered this question have an equivalent of a master degree. The other represented category were people that made a 2 year training after their high school degree (BAC). Some of the respondents had though a PHD degree.
+
+<Chart id="profile-q-4" pie title={false} />
+
+### How many years have you been coding professionally?
+
+The majority of the responses were from people that are coding for less than a year. The second group of people are people that were coding from 1 to 3 years. The majority of the people that answered this survey are therefore junior developers
```

Comment:
I think this "The majority of the people that answered this survey are therefore junior developers" is a wide claim as we cannot define the seniority level of a person based only on the number of years of experience. I suggest to change to something like "Based on this data we can assume a big majority of the people who answered this survey are therefore junior developers. It's however important to note that Seniority is not only based on the numbers of years of experience"